packages/ui/src/themes/full.ts

```diff
@@ -0,0 +1,6 @@
+import { visualStyle as alertVisualStyle } from '~/primitives/alert';
+import { buildTheme, mergeTheme } from './buildTheme';
+import { layoutTheme } from './layout';
+
+const visualTheme = buildTheme({ ...alertVisualStyle });
```

Comment:
So this will become a large spread of all of our internal appearance objects right?

Reply:
Yup, this was in order to support type checking the argument to `buildTheme` since the individual objects have a `satisfies` constraint that lets them retain their individual keys. When we drop the "complete theme" requirement we won't have to spread the objects.